automation/publish_show.py

```diff
@@ -0,0 +1,54 @@
+"""Script to be ran by GH Actions to build the archive and schedule the newsletter"""
+
+import typer
+from src.github import Episode
+from src import newsletter
+from src.engine import engine
+from dateutil import parser
+import pathlib
+
+def build_website(episode:Episode) -> str:
+    """Renders the content for the website"""
+    template = engine.get_template("archive.md")
+    github_filename = parser.isoparse(episode.created_at).strftime("%Y-%m-%d")
+    github_path = pathlib.Path(f"./site/content/{github_filename}.md")
+    content = template.render(
+        title=episode.title,
+        date=episode.created_at,
+        issues=episode.issues,
+        cfps=episode.cfps,
+        conferences=episode.conferences,
+        podcast=episode.podcast,
+        youtube=episode.youtube,
+        github=f"https://github.com/kjaymiller/Python-Community-News/blob/main/app/content/{{github_filename}}.md"
```

Comment:
```suggestion
        github=f"https://github.com/kjaymiller/Python-Community-News/blob/main/site/content/{{github_filename}}.md"
```

`app` has been renamed to `site`, correct?

Reply:
in this case yes and that's something I caught in the update. The new PR will have that changed